lib/maths/time_series/CTimeSeriesModel.cc

```diff
@@ -127,16 +123,10 @@ const std::string VERSION_7_11_TAG("7.11");
 const std::string ID_6_3_TAG{"a"};
 const std::string IS_NON_NEGATIVE_6_3_TAG{"b"};
 const std::string IS_FORECASTABLE_6_3_TAG{"c"};
-//const std::string RNG_6_3_TAG{"d"}; Removed in 6.5
 const std::string CONTROLLER_6_3_TAG{"e"};
 const core::TPersistenceTag TREND_MODEL_6_3_TAG{"f", "trend_model"};
 const core::TPersistenceTag RESIDUAL_MODEL_6_3_TAG{"g", "residual_model"};
 const std::string ANOMALY_MODEL_6_3_TAG{"h"};
-
-//const std::string RECENT_SAMPLES_6_3_TAG{"i"}; Removed in 6.5
-//const std::string CANDIDATE_CHANGE_POINT_6_3_TAG{"j"}; Removed in 7.11
-//const std::string CURRENT_CHANGE_INTERVAL_6_3_TAG{"k"}; Removed in 7.11
-//const std::string CHANGE_DETECTOR_6_3_TAG{"l"}; Removed in 7.11
```

Comment:
I don't think these should be deleted yet.  8.x still has to be able to load state going back to 6.3, so it's good to know what tags were used in that.  Even though they're obviously ignored because they're commented out they still serve a useful purpose as a reminder that a state might get restored that discards these.

Reply:
See [44ddc14](https://github.com/elastic/ml-cpp/pull/2270/commits/44ddc14320809228500abdbccb5c82a3c3eecdf5).